inc/template-tags.php

```diff
@@ -114,7 +114,7 @@ function twentyseventeen_edit_link( $id ) {
 	} elseif ( get_post( $id ) ) {
 		$type = __( 'Post', 'twentyseventeen' );
 	}
-	$link = edit_post_link(
```

Comment:
I tried to create new pull request by creating branch but failed. If you can take care of this pull request, may be I am able to create a new pull request.